src/pages/benefits/benefitFormUI/BenefitFormUI.tsx

```
@@ -114,16 +147,21 @@ const BenefitFormUI: React.FC = () => {
   };
   const handleFormSubmit = async () => {
     setDisableSubmit(true);
+
     const formDataNew: any = { ...formData };
-    formDataNew.benefit_id = id;
+
+    formDataNew.benefitId = id;
     delete formDataNew.docs;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)